src/app/services/sms/sms.service.ts

```diff
@@ -186,15 +186,12 @@ const sendSms = (
     smsType,
   }
 
-  const statusCallbackRoute = '/api/v3/notifications/twilio'
```

Comment:
the important part of this PR seems to have been removed?

Reply:
Seems like. I didnt handle some of the merge issues after pulling develop, will fix it!